styles/bootstrap.less

```diff
@@ -256,3 +256,6 @@ footer[role="contentinfo"] {
 .form-control-required {
   color: @brand-danger;
 }
+
+.item.references a {
+word-break: break-all;
```

Comment:
It looks like the syntax here is broken. You're missing a closing `}`.

Also, `word-break: break-all` will cause every word to split, even if it is not too long. Here are the properties Sophy used for the docs hub:

https://github.com/pkp/pkp-docs/pull/263/files

So I'd recommend something like this:

```
.references a {
  // sophy's properties/values
}
```